core/templates/dev/head/services/ExplorationHtmlFormatterService.js

```diff
@@ -23,21 +23,35 @@ oppia.factory('ExplorationHtmlFormatterService', [
   '$filter', 'extensionTagAssemblerService', 'HtmlEscaperService',
   function($filter, extensionTagAssemblerService, HtmlEscaperService) {
     return {
+      /**
+       * @param {string} interactionId - The id of interaction in camel case.
```

Comment:
Not sure that "camel case" is the best description, is it just the interaction id as-is?

Reply:
Yes, it is will change that.